rocky/katalogus/client.py

```diff
@@ -205,6 +209,15 @@ def get_cover(self, boefje_id: str) -> BytesIO:
         response.raise_for_status()
         return BytesIO(response.content)
 
+    def create_plugin(self, plugin: Plugin) -> None:
+        response = self.session.post(
+            f"{self.organization_uri}/plugins",
+            headers={"Content-Type": "application/json"},
+            content=plugin.model_dump_json(exclude_none=True),
+        )
+        logger.info("Created new plugin %s", plugin.name)
+        response.raise_for_status()
```

Comment:
Unless the `plugin` instance is not serializable by the default JSON serializer, the correct way to do this in `httpx` would be: `self.session.post('<endpoint>', json=plugin.model_dump(exclude_none=True))`. The `json` argument serializes the argument to JSON and sets the correct content type